src/Stratis.Bitcoin.IntegrationTests/ProofOfStakeSteps.cs

```diff
@@ -74,10 +74,17 @@ public void PremineNodeStartsStaking()
 
         public void PremineNodeWalletHasEarnedCoinsThroughStaking()
         {
+            var network = this.nodes[this.PremineNode].FullNode.Network;
+            var premine = network.Consensus.ProofOfWorkReward + network.Consensus.PremineReward;
+            var mineToMaturity = network.Consensus.ProofOfWorkReward * 110;
+            var balanceShouldBe = premine + mineToMaturity;
+
+            var spendable = this.nodes[this.PremineNode].FullNode.WalletManager().GetSpendableTransactionsInWallet(this.PremineWallet, includeImmature: true);
```

Comment:
I know this is just as for which we don't have many requirements in terms of code style, so just saying that about five lines should not use var